src/github.com/getlantern/flashlight/client/config.go

```diff
@@ -18,9 +18,8 @@ type ClientConfig struct {
 	DumpHeaders    bool // whether or not to dump headers of requests and responses
 	FrontedServers []*FrontedServerInfo
 	ChainedServers map[string]*ChainedServerInfo
+	ProxiedSites   *proxiedsites.Config // List of proxied site domains that get routed through Lantern rather than accessed directly
```

Comment:
I moved this so that ProxiedSites comes before MasqueradeSets in the yaml.  It's more interesting and more likely to be edited by the user, so this is a nice convenience.